utility/doc/TRUSTLESS_RELAY_VALIDATION.md

```diff
@@ -0,0 +1,73 @@
+# Validation of Trustless Relays
+
+- [A. Client-side Relay Validation](#a-client-side-relay-validation)
+- [B. Server-side Relay Validation](#b-server-side-relay-validation)
+
+## A. Client-side Relay Validation
+
+When an application requests to send a trustless relay, the CLI performs several checks on the relay before sending it to the specified servicer.
+The following diagram lists all these checks with links to the corresponding code secion (or an issue if the check is not implemented yet).
+
+```mermaid
+---
+title: Client-side Relay Validation
+---
+graph TD
+    app_key{<b><a href='https://github.com/pokt-network/pocket/blob/bfc57eb89907932831b5730c68a09b4656c420c2/app/client/cli/servicer.go#L91'>Validate app key</a></b>}
```

Comment:
Huge fan of the diagrams and links to the code. Nice and simple w/ great signposting 💯